dbgpt/rag/transformer/graph_embedder.py

```diff
@@ -0,0 +1,51 @@
+"""GraphEmbedder class."""
+
+import logging
+from typing import List
+
+from dbgpt.rag.transformer.text2vector import Text2Vector
+from dbgpt.storage.graph_store.graph import Graph, GraphElemType
+
+logger = logging.getLogger(__name__)
+
+
+class GraphEmbedder(Text2Vector):
+    """GraphEmbedder class."""
+
+    def __init__(self):
+        """Initialize the GraphEmbedder"""
+        super().__init__()
+    
+    async def embed(
+        self,
+        text: str,
+    ) -> List[float]:
+        """Embed"""
+        return await super()._embed(text)
+            
+    async def batch_embed(
+        self,
+        graphs_list: List[List[Graph]],
+    ) -> List[List[Graph]]:
+        """Embed graphs from graphs in batches"""
+
+        for graphs in graphs_list:
+            for graph in graphs:
+                for vertex in graph.vertices():
+                    if vertex.get_prop("vertex_type") == GraphElemType.CHUNK.value:
+                        text = vertex.get_prop("content")
+                        vector = await self._embed(text)
+                        vertex.set_prop("embedding", vector)
+                    elif vertex.get_prop("vertex_type") == GraphElemType.ENTITY.value:
+                        vector = await self._embed(vertex.vid)
+                        vertex.set_prop("embedding", vector)
```

Comment:
Vid of the entity is the keywords of the entity. We use the vec of it to search the similiar keywords instead of using the keywords matching directly.